contracts/Everlong.sol

```diff
@@ -399,22 +462,59 @@ contract Everlong is IEverlong {
 
     // TODO: Decide if we want to put a slippage guard here.
     //
+    /// @notice Closes mature positions in the Everlong portfolio.
+    /// @param _limit Limit the maximum number of positions to close.
+    ///               - Zero indicates no limit.
+    /// @return output Amount of assets received from the closed positions.
+    function closeMaturedPositions(
+        uint256 _limit
+    ) external returns (uint256 output) {
+        output = _closeMaturedPositions(_limit);
+    }
```

Comment:
Same here, what's the rationale for having two functions? We can just make this `public` if we want this to be accessible externally.

Reply:
Leftovers, intially had it guarded with `onlyAdmin` but decided against it and forgot to clean up